pkg/genevalogging/genevalogging.go

```diff
@@ -5,6 +5,7 @@ package genevalogging
 
 import (
 	"context"
+	"crypto/sha1"
```

Comment:
we're audited on our choice of cryptographic functions.  It makes no odds here but use sha256 please